packages/client/src/components/apps/createForm.tsx

```diff
@@ -83,9 +83,9 @@ const autoGenId = (name: string) => {
 class AppCreateForm extends React.Component<Props, State> {
   constructor(props) {
     super(props);
-    const { phAppTemplates, templateId } = props;
+    const { phAppTemplates, templateId, initialValue, type } = props;
     const currentTemplate = find(phAppTemplates, v => v.id === templateId);
-    const defaultEnvs = currentTemplate ? currentTemplate.defaultEnvs : [];
+    const defaultEnvs = currentTemplate ? currentTemplate.defaultEnvs : type === 'edit' ? initialValue.appDefaultEnv : [];
```

Comment:
I think we should get defaultEnvs from initialValue if initialVale exists (editing mode) because defaultEnvs of template may have been changed even it has the same template id.